src/components/checkbox-group/checkbox/bl-checkbox.ts

```diff
@@ -177,6 +177,13 @@ export default class BlCheckbox extends FormControlMixin(LitElement) {
   private handleFieldValueChange = (event: CustomEvent<Array<string>>) => {
     this.checked = event.detail.includes(this.value);
   };
+  private handleKeyDown(event: KeyboardEvent) {
+    if (event.code === "Enter" || event.code === "Space") {
```

Comment:
I believe we should avoid using the Enter key for checking

Reply:
Yes, thank you. Enter key is not supported in checkbox accessibility. I missed that part.